ks/azure/centos68-hpc.ks

```diff
@@ -0,0 +1,183 @@
+
+# Clear the MBR
+zerombr
+
+# Disk partitioning information
+part / --fstype="ext4" --size=1 --grow --asprimary
+
+# System bootloader configuration
+bootloader --location=mbr --append="numa=off console=ttyS0,115200n8 earlyprintk=ttyS0,115200 rootdelay=300"
+
+# Add OpenLogic repo
+repo --name=openlogic --baseurl=http://olcentgbl.trafficmanager.net/openlogic/6/openlogic/x86_64/
+
+# Firewall configuration
+firewall --disabled
+
+# Enable SELinux
+selinux --enforcing
+
+# Don't configure X
+skipx
+
+# Power down the machine after install
+poweroff
+
+%packages
+@base
+@console-internet
+@core
+@debugging
+@directory-client
+@hardware-monitoring
+@java-platform
+@large-systems
+@network-file-system-client
+@performance
+@perl-runtime
+@server-platform
+ntp
+dnsmasq
+cifs-utils
+sudo
+python-pyasn1
+parted
+WALinuxAgent
+msft-rdma-drivers
+kernel-headers
+-hypervkvpd
+-dracut-config-rescue
+
+%end
+
+%post --log=/var/log/anaconda/post-install.log
+
+#!/bin/bash
+
+# Disable the root account
+usermod root -p '!!'
+
+# Remove unneeded parameters in grub
+sed -i 's/ numa=off//g' /boot/grub/grub.conf
+sed -i 's/ rhgb//g' /boot/grub/grub.conf
+sed -i 's/ quiet//g' /boot/grub/grub.conf
+sed -i 's/ crashkernel=auto//g' /boot/grub/grub.conf
+
+# Set OL repos
+curl -so /etc/yum.repos.d/CentOS-Base.repo https://raw.githubusercontent.com/szarkos/AzureBuildCentOS/master/config/azure/CentOS-Base.repo
+curl -so /etc/yum.repos.d/OpenLogic.repo https://raw.githubusercontent.com/szarkos/AzureBuildCentOS/master/config/azure/OpenLogic.repo
+
+# Import CentOS and OpenLogic public keys
+curl -so /etc/pki/rpm-gpg/OpenLogic-GPG-KEY https://raw.githubusercontent.com/szarkos/AzureBuildCentOS/master/config/OpenLogic-GPG-KEY
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
+rpm --import /etc/pki/rpm-gpg/OpenLogic-GPG-KEY
+
+# Modify yum
+echo "http_caching=packages" >> /etc/yum.conf
+
+# Enable SSH keepalive
+sed -i 's/^#\(ClientAliveInterval\).*$/\1 180/g' /etc/ssh/sshd_config
+
+# Changing password retrictions defined by CIS CentOS Linux 6 Benchmark
+sudo sed -i 's/pam_cracklib.so try_first_pass retry=3 type=/\pam_cracklib.so try_first_pass retry=3 minlen=14 dcredit=-1 ucredit=-1 ocredit=-1 lcredit=-1/g' /etc/pam.d/system-auth
+
+# Configure network
+cat << EOF > /etc/sysconfig/network-scripts/ifcfg-eth0
+DEVICE=eth0
+ONBOOT=yes
+BOOTPROTO=dhcp
+TYPE=Ethernet
+USERCTL=no
+PEERDNS=yes
+IPV6INIT=no
+EOF
+
+cat << EOF > /etc/sysconfig/network
+NETWORKING=yes
+HOSTNAME=localhost.localdomain
+EOF
+
+# Disable persistent net rules
+touch /etc/udev/rules.d/75-persistent-net-generator.rules
+rm -f /lib/udev/rules.d/75-persistent-net-generator.rules /etc/udev/rules.d/70-persistent-net.rules
+
+# Disable some unneeded services by default (administrators can re-enable if desired)
+chkconfig cups off
+
+# Enable RDMA driver
+
+  ## Need to pull these packages from latest 6.8 repo
+  yum -y install rdma librdmacm libmlx4 dapl libibverbs
+
+  ## Install LIS4.1 with RDMA drivers
```

Comment:
We'll need to look at this section closer and make sure to point to /opt/microsoft/rdma/rhel68 instead of rhel65.  We'll also need to change the kernel version from -431 to -642